extras/jackson-support/src/test/java/com/palantir/remoting1/ext/jackson/ObjectMappersTest.java

```diff
@@ -52,7 +53,33 @@ public void serializeJdk7ModuleObject() {
     }
 
     @Test
+    public void deserializeJdk8ModuleObject() {
+        String jsonPresent = "\"Test\"";
+        String jsonNone = "null";
+
+        try {
+            assertThat(MAPPER.readValue(jsonPresent, Optional.class)).isEqualTo(Optional.of("Test"));
+            assertThat(MAPPER.readValue(jsonNone, Optional.class)).isEqualTo(Optional.empty());
+        } catch (IOException e) {
```

Comment:
just have test `throw IOException` as this will lose any useful context on exception